lib/Minz/Session.php

```diff
@@ -35,13 +68,34 @@ public static function param($p, $default = false) {
 	 * @param $v la valeur à attribuer, false pour supprimer
 	 */
 	public static function _param($p, $v = false) {
+		if (!self::$volatile && !self::$locked) {
+			session_start();
```

Comment:
If we wanted to get fresh values from a potential concurrent session, we would need to do the same for `param()`.
I am not sure this is desirable though